schemas/learningResource.schema.tpl.json

```diff
@@ -0,0 +1,49 @@
+{
+  "_type": "https://openminds.ebrains.eu/publications/LearningResource",
+  "_extends": "creativeWork.schema.tpl.json",
+  "required": [
+    "name",
+    "about",
+    "datePublished"
+  ],
+  "properties": {
+    "competencyRequired": {
+        "type": "string",
+        "_instruction": "Enter the knowledge, skills, or abilities that are prerequisites for being able to use the learning resource."
+    },
+    "educationalLevel": {
```

Comment:
I think that should become a controlledTerm terminology.

Reply:
probably, yes. The problem is there are so many possible terminologies, e.g. ["beginner", "intermediate", "advanced"], ["undergraduate", "master", "doctoral", "post-graduate"], ["A1", "A2", "B1", "B2", "C1", "C2"]

Reply:
I've now made this a controlled term

Reply:
okay. as for what to put in there: I would stick to beginner intermediate and advanced here. All the other terms might be often used in the communities but they always require interpretation that could lead to misunderstandings (an bio post doc might be still a beginner in an informatics problem)